linked_list/linked_list.py

```diff
@@ -1,79 +1,158 @@
 
 # Defines a node in the singly linked list
+from operator import truediv
+
 class Node:
 
-    def __init__(self, value, next_node = None):
+    def __init__(self, value, next_node = None, previous_node = None):
         self.value = value
         self.next = next_node
+        self.previous = previous_node

```

Comment:
Usually we only maintain a previous node if we have a doubly linked list, in which you would also want to maintain a tail pointer in the `LinkedList` constructor